bootstrap/kubeadm/internal/cloudinit/cloudinit_test.go

```diff
@@ -94,9 +95,12 @@ func TestNewInitControlPlaneCommands(t *testing.T) {
 
 	cpinput := &ControlPlaneInput{
 		BaseUserData: BaseUserData{
+			BootCommands: []bootstrapv1.BootCommand{
+				{"echo", "$(date) hello BootCommands!"},
```

Comment:
Question: can't this be the same as below?

```suggestion
				{`"echo $(date) ': hello BootCommands!'"`},
```

Reply:
I wanted to test the output when having multiple commands, but it didn't make sense to split it there.

This should be clearer https://github.com/kubernetes-sigs/cluster-api/pull/11271/commits/1874ba875fa80ef4afb1fe11786cb69d1829136d